keps/sig-storage/625-csi-migration/kep.yaml

```diff
@@ -14,7 +14,35 @@ approvers:
   - "@saadali"
 editor: "@davidz627"
 creation-date: 2019-01-29
-last-updated: 2019-01-29
+last-updated: 2021-02-03
+disable-supported: true
 status: implementable
 see-also:
   - "https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/csi-migration.md"
+prr-approvers:
+  - "@wojtek-t"
+replaces:
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: beta
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.17"
```

Comment:
1.17 does seem correct here (nor below)

Reply:
Hmm, sorry what do you mean here? I want to clarify that CSI migration feature has been Beta since v1.17. So it seems v1.17 is the right version to put here?

Reply:
What's confusing is we have one overall kep for csi migration, but each cloud provider has their own feature gates and timelines. Some drivers went beta in 1.17, but for 1.21, we are targeting AzureFile beta. So maybe we have "latest-milestone" as 1.21, but keep beta at "1.17". And in the implementation history section, call out the timelines of each cloud provider.

Reply:
Good point, I add a new section in implementation history.